### ISSUE TITLE
Umlaut leads to crash

### ISSUE THREAD

**Issue by ajerni** [rparrett/bevy_simple_text_input#44]
version 0.5.1 panics when I enter an "umlaut" lie ä, ö, ü - is this a bug here or some kind of global bevy setting?

**Comment by rparrett** [comment]
Could you please test the main branch? https://github.com/rparrett/bevy_simple_text_input/pull/42 is included there, but hasn't been released yet because I'm not confident that it has been tested thoroughly enough.

**Comment by ajerni** [comment]
beginner here, sorry :-) How can I test the main branch? I have `bevy_simple_text_input = "0.5.1"` in my Cargo.toml

**Comment by rparrett** [comment]
```
bevy_simple_text_input = { git = "https://github.com/rparrett/bevy_simple_text_input.git" }
```
Should do the trick.

**Comment by ajerni** [comment]
Thanks a lot. Yes, it does the trick: it does not panic anymore - but the ä ö or ü are not displayed but instead a "square" is shown as a character.

**Comment by rparrett** [comment]
> but the ä ö or ü are not displayed but instead a "square" is shown as a character.

Cool. That's expected with Bevy's default font, if that's what you're using. It is a very basic font.

**Comment by rparrett** [comment]
Released 0.6.0.
